src/tests/roc_audio/test_freq_estimator.cpp

```diff
@@ -21,7 +21,7 @@ const double Epsilon = 0.0001;
 
 } // namespace
 
-TEST_GROUP(freq_estimator) {};
+TEST_GROUP(freq_estimator){};
```

Comment:
I think that this change does not relate to the current PR.

Very rarely some files can be pushed to the develop branch without the required formatting. As a result, next time you run `scons fmt -Q` you will see these files to be formatted. In such case I would suggest to reset these unrelated changes and commit only changes related to the current PR.

Reply:
See my comment about formatting above.

Reply:
Yeah I was definitely using an older version,  but I shouldn't have included these files in my PR.  I'll limit my scope next time